app/src/Components/Video.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react'
+import VideoComponent from 'react-native-video'
+import styled from 'styled-components/native'
+
+interface VideoProps {
+    details: string
```

Comment:
As this is a URI/URL could we name it `uri` or `url` rather than `details` so it's clearer what it is?